packages/icons/icons.json

```
@@ -2606,161 +2606,161 @@
         "group": "action",
         "codepoint": 61847
     },
     {
         "displayName": "Follower",
         "iconName": "follower",
-        "tags": "person, human, male, female, character, customer, individual, social",
+        "tags": "person, human, male, female, character, customer, individual, social, share, add, send, move, delegate, move person, next",
         "group": "interface",
         "codepoint": 61885
     },
     {
         "displayName": "Following",
         "iconName": "following",
-        "tags": "person, human, male, female, character, customer, individual, social",
+        "tags": "person, human, male, female, character, customer, individual, social, receive, message, request",
         "group": "interface",
         "codepoint": 61886
     },
     {
         "displayName": "Menu",
         "iconName": "menu",
-        "tags": "navigation, lines, list",
+        "tags": "navigation, lines, list, stack, hamburger, paragraph, document, section",
         "group": "interface",
         "codepoint": 62016
     },
     {
         "displayName": "Collapse all",
         "iconName": "collapse-all",
-        "tags": "arrows, chevron, reduce",
+        "tags": "arrows, chevron, reduce, smash, in between, up down",
         "group": "interface",
         "codepoint": 61790
     },
     {
         "displayName": "Expand all",
         "iconName": "expand-all",
-        "tags": "arrows, chevron, enlarge",
+        "tags": "arrows, chevron, enlarge, stretch, flex, volume",
         "group": "interface",
         "codepoint": 61855
     },
     {
         "displayName": "Intersection",
         "iconName": "intersection",
-        "tags": "circles, combine, cross",
+        "tags": "circles, combine, cross, venn diagram, overlap, link, intersect, union, inner join",
         "group": "action",
         "codepoint": 61957
     },
     {
         "displayName": "Blocked person",
         "iconName": "blocked-person",
-        "tags": "person, human, male, female, character, customer, individual, social, banned, prohibited",
+        "tags": "human, male, female, character, customer, individual, social, banned, prohibited, box remove, box minus",
         "group": "interface",
         "codepoint": 61748
     },
     {
         "displayName": "Slash",
         "iconName": "slash",
-        "tags": "divide, separate",
+        "tags": "divide, separate, backslash, or",
         "group": "action",
         "codepoint": 62137
     },
     {
         "displayName": "Percentage",
         "iconName": "percentage",
-        "tags": "modulo, modulus",
+        "tags": "modulo, modulus, fraction, statistics",
         "group": "action",
         "codepoint": 62065
     },
     {
         "displayName": "Satellite",
         "iconName": "satellite",
-        "tags": "communication, space",
+        "tags": "communication, space, radar, sensor, airplane, wifi, landing",
```

```suggestion
        "tags": "communication, space, radar, sensor, airplane, data, gps, orbit, landing",
```

---

I like wifi (unless we have a separate wifi icon which i dont think we do)
This would be a good icon option if i wanted to represent wifi
I do like your other additions though!

---

wifi almost always comes from land-based signals, so I consider that distinct from satellite-enabled communication.

it is unfortunate that we don't have a good "wifi" icon. I still don't think "satellite" is the best one for that, maybe "antenna" is better. but I suppose in the meantime while we don't have a good "wifi" icon we can leave this tag here.

```suggestion
        "tags": "communication, space, radar, sensor, airplane, wifi, data, gps, orbit, landing",
```